File: app/src/main/java/com/anokmik/tripassistant/util/contract/view/ViewListItem.java
package com.anokmik.tripassistant.util.contract.view;

import android.support.annotation.LayoutRes;

public interface ViewListItem {

    @LayoutRes
    int getRowItemLayoutId();

    int getItemBindingId();

    int getAdapterPositionProviderBindingId();

    int getItemListenerBindingId();

    int getItemIsEditingBindingId();

}

File: app/src/main/kotlin/com/anokmik/tripassistant/util/contract/view/ViewListItem.kt
package com.anokmik.tripassistant.util.contract.view

interface ViewListItem {

    val rowItemLayoutId: Int

    val itemBindingId: Int

    val adapterPositionProviderBindingId: Int

    val itemListenerBindingId: Int

    val itemIsEditingBindingId: Int

}